include/random_walks/gaussian_hamiltonian_monte_carlo_exact_walk.hpp

```diff
@@ -10,10 +10,17 @@
 
 #include "sampling/sphere.hpp"
 
+#include <iostream>
 
+typedef long double LNT;
+typedef double NT;
```

Comment:
NT is used for the template parameter of Number Type, it is confusing if you define an alias here.